engine/pokedex/pokedex_2.asm

```diff
@@ -124,14 +124,16 @@ DisplayDexEntry:
 	jr z, .skip_height
 	push hl
 	push de
+; Print the height with two of the four digits will be in front of the decimal point
```

Comment:
Grammar: "Print the height, with two of the four digits in front of the decimal point."